src/nanoemoji/write_font.py

```diff
@@ -257,16 +259,22 @@ def _create_glyph(color_glyph: ColorGlyph, painted_layer: PaintedLayer) -> Glyph
     return glyph
 
 
-def _draw_glyph_extents(ufo: ufoLib2.Font, glyph: Glyph):
+def _draw_glyph_extents(
+    ufo: ufoLib2.Font, glyph: Glyph, bounds: Optional[Tuple[float, float, float, float]]
+):
     # apparently on Mac (but not Linux) Chrome and Firefox end up relying on the
     # extents of the base layer to determine where the glyph might paint. If you
     # leave the base blank the COLR glyph never renders.
 
-    # TODO we could narrow this to bbox to cover all layers
+    start = (0, 0)
+    end = (ufo.info.unitsPerEm, ufo.info.unitsPerEm)
```

Comment:
We should have bounds unless nothing at all is drawn, correct? If so, I think just not drawing anything in the base glyph would make sense, e.g. if bounds.area == 0: exit

Reply:
made it to return early if rectangle area is 0 (unlikely but you never know)